src/components/Grid.tsx

```diff
@@ -546,11 +546,16 @@ class Grid extends React.Component<
               </button>
               : null}
             {/* Generate a new box for sorting options */}
-            <h2 className="marketplace-header__label">{t("grid.sort.label")}</h2>
-            <SortBox
-              onChange={(value) => this.updateSort(value)}
-              sortBoxOptions={generateSortOptions(t)}
-              sortBySelectedFn={(a) => a.key === this.CONFIG.sort} />
+            {this.CONFIG.activeTab !== "Snippets" ? (
+              <>
+                <h2 className="marketplace-header__label">{t("grid.sort.label")}</h2>
+                <SortBox
+                  onChange={(value) => this.updateSort(value)}
+                  sortBoxOptions={generateSortOptions(t)}
+                  sortBySelectedFn={(a) => a.key === this.CONFIG.sort} />
+              </>
+            ) : <h1>{this.props.title}</h1>
```

Comment:
Is the `this.props.title` the "Spicetify Marketplace text? I removed that in the original PR.

Reply:
> I removed that in the original PR

I know, but without it header looks empty

Reply:
Still, doesn't make sense to have a title only show up on a single tab.